controllers/storagecluster/storageclasses.go

```diff
@@ -395,6 +436,12 @@ func (r *StorageClusterReconciler) newStorageClassConfigurations(initData *ocsv1
 		newCephFilesystemStorageClassConfiguration(initData),
 		newCephBlockPoolStorageClassConfiguration(initData),
 	}
+	// If openshift virtualization is installed with the kubevirt crd, We create a specialized storageclass for it.
```

Comment:
I am not familiar with this operator but is this function called from within a reconcile loop?  In the case where kubevirt is installed after this operator will the ceph-rbd-virtualization SC be created?

Reply:
We typically recommend that storage be configured before installing kubevirt so that a default storage class can be set.

Reply:
We have added a watch on the kubevirt crd in our controller. So if at any point the kubevirt crd comes up the controller will fire up a reconcile & the storageclass will be immediately created